tests/test_interface.rb

```diff
@@ -165,6 +173,9 @@ def validate_get_switchport(inttype_h)
                    "Error: #{interface.name}, switchport mode, default, " \
                    'not correct')
     end
+  rescue => e
```

Comment:
Sure - I interpreted the spec as saying that the default exception is RuntimeError but that doesn't appear to be the case so I'll fix it.